mergify_engine/branch_updater.py

```diff
@@ -119,7 +120,7 @@ async def _do_rebase(ctxt: context.Context, user: user_tokens.UserTokensUser) ->
     git = gitter.Gitter(ctxt.log)
     try:
         await git.init()
-        if ctxt.subscription.active:
+        if subscription.Features.PUBLIC_REPOSITORY:
```

Comment:
To check a feature you need to use the `ctxt.subscription.has_feature()` method.

The feature to check here need to be passed down from the actions (rebase/merge/queue) because is can be BOT_ACCOUNT or UPDATE_BOT_ACCOUNT depending on which action call this fonction.